src/metabase/lib/metadata/ident.cljc

```diff
@@ -48,3 +48,17 @@
    (map #(attach-ident (prefix-fn (:table-id %)) %)))
   ([prefix-fn columns]
    (sequence (attach-idents prefix-fn) columns)))
+
+(defn implicitly-joined-ident
+  "Returns the ident for an implicitly joined column, given the idents of the foreign key column and the target column.
+
+  Remember that `:ident` strings should never be parsed - they are opaque, but should be legible during debugging."
+  [fk-ident target-ident]
+  (str "implicit_via__" fk-ident "__->__" target-ident))
```

Comment:
refreshing my memory: these `fk-ident` are something like `db__schema__table__field_name`, no?

Reply:
Probably: fields get idents like that, so FKs etc. will probably look like that.

I'm not sure if there's a concern or follow-up question here.

Reply:
no concern